bazel/action.yml

```diff
@@ -114,6 +118,11 @@ runs:
       shell: bash
       run: echo "BAZEL_FLAGS=$BAZEL_FLAGS --repository_cache=$(pwd)/${{ env.REPOSITORY_CACHE_PATH }}" >> $GITHUB_ENV
 
+      # TODO(b/467211650)
```

Comment:
To be more specific, something like:
TODO(b/467211650) this shouldn't be necessary, remove it